geosnap/harmonize/harmonize.py

```diff
@@ -1,5 +1,6 @@
 import pandas as pd
 import geopandas as gpd
+from warnings import warn
```

Comment:
Where is warn being used?

Reply:
good catch. i swapped it out later for the ValueError